lib/include/prjxray/xilinx/bitstream_reader.h

```diff
@@ -93,8 +93,7 @@ class BitstreamReader {
 
```

Comment:
Why are there changes to this file?

Reply:
Hi @rw1nkler , I see CI had started to fail due to formatting after you rebased on main.   Did the format issue come from main?  And if so, how did it get there?   Or is it a new formatting check?

Reply:
This is due to the wrong formatting merged in the current master.
I will prepare a separate PR with the changes.

Reply:
https://github.com/SymbiFlow/prjxray/pull/1513